gmso/formats/networkx.py

```diff
@@ -0,0 +1,953 @@
+import matplotlib.pyplot as plt
+import networkx as nx
+import unyt
+
+from ipywidgets import interact, fixed
+import ipywidgets as widgets
+
+from gmso.external.convert_networkx import to_networkx
+
+
+def plot_networkx_params(networkx_graph, list_of_edges):
+    """Get a networkx plot showing the specified edges in a networkx graph object.
+
+    Parameters
+    ----------
+    networkx_graph : A networkx.Graph object
+        This is a networkx graph object that can be created from a topology using the to_networkx
+        function found in gmso.external.convert_networkx
+    list_of_edges : a list of edges that should be shown on the plot
+        Will be of the shape [(node1,node2), (node2,node3), etc...]
+
+    Returns
+    -------
+    matplotlib.pyplot.figure
+        The drawn networkx plot of the topology on a matplotlib editable figures
+
+    matplotlib.pyplot.axis
+        The axis information that corresponds to a networkx drawn figure. This output can be
+        shown using
+        matplotlib.pyplot.show()
+    """
+    fig, ax = plt.subplots(1, 1, figsize=(8, 8))
+    edge_weights, edge_colors = highlight_networkx_edges(networkx_graph, list_of_edges)
+    ax = plot_networkx_nodes(
+        networkx_graph, ax, edge_weights=edge_weights, edge_colors=edge_colors
+    )
+
+    return (fig, ax)
+
+
+def interactive_networkx_atomtypes(topology, list_of_labels=None):
+    """Get an interactive networkx plot showing the atom types of a topology object.
+
+    Parameters
+    ----------
+    topology : A gmso.core.topology.Topology object
+        This should be a gmso topology object that you want to visualize the atom types
+        that have been parameterized.
+    list_of_labels : Additional labels you would like to have on the plot.
+        Default labels are ['atom_type.name','charge','mass','element','label','position'].
+        Any additonal labels can be appended from the list of:
+        `dir(list(topology.sites)[0])` or `dir(list(topology.sites)[0].atom_type)`
+
+    Returns
+    -------
+    ipywidgets.Dropdown()
+        Two dropdown options, corresponding to:
+            Label = What labels are shown on the networkx graph
+            Atom_Name = Which sites will show these labels
+
+    matplotlib.pyplot.figure
+        The drawn networkx plot of the topology on a matplotlib editable figures
+
+    matplotlib.pyplot.axis
+        The axis information that corresponds to that figure. This output can be
+        shown using
+        matplotlib.pyplot.show()
```

Comment:
Are there return values justified here?

Reply:
I wasn't quite sure how to handle this. It does plot the interactive ipywidget with these features, but doesn't explicitly return these objects. Is there a better way to annotate what running this function would give the user without using the return section of the docstring?